libs/hwServer/manufacturers/ledger/index.js

```diff
@@ -25,6 +26,7 @@ const addDevice = (device, path, { add }) => {
     model: device.productName,
     path,
     manufacturer: LEDGER.name,
+    status: DEVICE_STATUS.STAND_BY,
   };
```

Comment:
@soroushm Do you know why we do Math.random() for `deviceId` here?
It will look pretty strange for the user if the status in the TopBar gets a new id each time they connect or disconnect their device. 

Also, to be able to complete https://github.com/LiskHQ/lisk-desktop/issues/4769,  we should add a `modelName: device.product`.

This probably need to change to: 
```
const addDevice = (deviceModel, device { add }) => {
  const newDevice = {
    productId: device.productId, // '4113' Which is the ID we show in the top bar
    modelName: device.product, // 'Nano S' Which is the model we show in the top bar
    path: device.path,
    ... (keeping the old values as well)
  };
```

Reply:
For reference, this is what the TransportNodeHid.listener returns
```
  TransportNodeHid.listen({
    next: (props) => {
      console.log(props);
      if (deviceModel && descriptor) {
        if (type === ADD_DEVICE) addDevice(deviceModel, descriptor, actions); // Here we need to update
        removeDevice(transport, actions);
      }
    },
  });
```
<img width="824" alt="image" src="https://user-images.githubusercontent.com/8784876/217776793-1633be30-944d-4bb6-8381-c6159caf5d91.png">

Reply:
@oskarleonard we can review it difrent ticket its out of the scope of current ticket

Reply:
Create this https://github.com/LiskHQ/lisk-desktop/issues/4799